### ISSUE TITLE
Pre-trained Model

### ISSUE THREAD

**Issue by andrianid** [maszhongming/MatchSum#8]
Hi, how to use the pre-trained model? Where should I put the torch.load() function?

**Comment by maszhongming** [comment]
If you want to use our pre-trained model as a component of your model, you can use `torch.load ()` in `__init__`.
If you want to test directly with our pre-trained model, you can refer to the test section in README, *save_path* should be the folder where you store the pre-trained model.

**Comment by andrianid** [comment]
Thank you. It can run well!

I also want to know if I want to process my own data, how can I get the "label" field?

**Comment by maszhongming** [comment]
`label` is the greedy extractive oracle we use to train our BertExt, the details can be found in this paper: SummaRuNNer: A Recurrent Neural Network based Sequence Model for Extractive Summarization of Documents, Page 3, Extractive Training section. Besides, there is no need to use the `label` field when training or testing MatchSum.
If you want to train BertExt model on your own data, you can refer to [PreSumm](https://github.com/nlpyang/PreSumm/tree/master/src), their code contains the part of extracting the label.
